### PR TITLE
API: Add `expand_dims` and `flip` functions for COO format

### PR DESCRIPTION
**Pull request by mtsokol**
Hi @hameerabbasi,

This PR adds `expand_dims` and `flip` Array API functions for the COO format to the main namespace.

### CODE REVIEW

File: sparse/__init__.py [hunk 1]
```diff
@@ -116,6 +116,7 @@
     std,
     sum,
     tensordot,
+    var,
```
Comment (hameerabbasi) [inline sparse/__init__.py:119]: This should also be added to `docs/generated/sparse.rst`.

**Review by hameerabbasi — CHANGES_REQUESTED** [review]
LGTM, just the one comment.

File: sparse/__init__.py (hunk 1 above)
Comment (mtsokol) [reply sparse/__init__.py:119]: Done!

**Review by hameerabbasi — APPROVED** [review]
LGTM, thanks for awesome work!

**Pull request merged**